docs/actions.md

```diff
@@ -2933,6 +2933,100 @@ Key | Description
 
 
 
+### automatic_code_signing
+
+Updates the Xcode 8 Automatic Codesigning Flag
+
+
+
+> Updates the Xcode 8 Automatic Codesigning Flag of all targets in the project
+
+automatic_code_signing | 
+-----|----
+Supported platforms | ios, mac
+Author | @mathiasAichinger, @hjanuschka
+Returns | The current status (boolean) of codesigning after modification
+
+
+
+<details>
+<summary>8 Examples</summary>
+
+```ruby
+# enable automatic code signing
+enable_automatic_code_signing
```

Comment:
it was in the plugin integration pr here: https://github.com/fastlane/fastlane/pull/8417/files#diff-b9ec8760f6bfcc35a81f1164b8cf0f5c

so a rebase on that PR should do it.